applications/GeoMechanicsApplication/tests/cpp_tests/test_T_micro_climate_flux_condition.cpp

```diff
@@ -406,9 +390,7 @@ KRATOS_TEST_CASE_IN_SUITE(CalculateLocalSystemForThermalMicroClimateCondition2D3
     p_condition->CalculateLocalSystem(lhs_matrix, rhs_vector, r_model_part.GetProcessInfo());
 
     auto expected_lhs_matrix = Matrix{3, 3, 0.0};
-    expected_lhs_matrix <<= 21.2568, -8.50271, 25.5081,
-            -8.50271, 12.7541, 8.50271,
-            25.5081, 8.50271, 68.0217;
+    expected_lhs_matrix <<= 21.2568, -8.50271, 25.5081, -8.50271, 12.7541, 8.50271, 25.5081, 8.50271, 68.0217;
```

Comment:
This might be nice to revert and add the tags to ignore this for formatting

Reply:
Done.